nodes/Soroban/actions/server/findLiquidityPool/execute.ts

```diff
@@ -0,0 +1,48 @@
+import { IExecuteFunctions } from 'n8n-workflow';
+import axios from 'axios';
+import { SorobanNetwork } from '../../../transport';
+import IAsset from '../../../../../common/interfaces/stellar/IAsset';
+import IAssetsPath from '../../../../../common/interfaces/stellar/IAssetsPath';
+import LiquidityPoolNotFoundError from '../../../../../common/errors/stellar/LiquidityPoolNotFoundError';
+
+export async function findLiquidityPool(this: IExecuteFunctions) {
+	const sorobanNetwork = await SorobanNetwork.setNetwork.call(this);
```

Comment:
Have you tried this and type it as a string?
```suggestion
	const { url } = await SorobanNetwork.setNetwork.call(this);
```